<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="directoryCollapsibleButton">
     <property name="text">
      <string>Directory Selection</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="directoryButton"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load NIfTI Files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="annotationCollapsibleButton">
     <property name="text">
      <string>Annotation Tools</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="placePointsButton">
        <property name="text">
         <string>Place Extreme Points</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="createBBoxButton">
        <property name="text">
         <string>Create Bounding Box</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="relaxLabel">
        <property name="text">
         <string>Bounding Box Relaxation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="relaxSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit Label</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>Next Sample</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>